src/test/scala/chiselTests/UIntOps.scala

```diff
@@ -172,15 +172,15 @@ class MatchedRotateLeftAndRight(w: Int = 13) extends BasicTester {
 }
 
 class UIntLitExtractTester extends BasicTester {
-  assert("b101010".U(2) === false.B)
-  assert("b101010".U(3) === true.B)
-  assert("b101010".U(100) === false.B)
+  assert("b101010".U.extract(2) === false.B)
+  assert("b101010".U.extract(3) === true.B)
+  assert("b101010".U.extract(100) === false.B)
   assert("b101010".U(3, 0) === "b1010".U)
   assert("b101010".U(9, 0) === "b0000101010".U)
 
-  assert("b101010".U(6.W)(2) === false.B)
-  assert("b101010".U(6.W)(3) === true.B)
-  assert("b101010".U(6.W)(100) === false.B)
+  assert("b101010".U(6.W).extract(2) === false.B)
```

Comment:
Why did these 3 tests need to be changed? I thought that if you put the (6.W) it *is* OK to put the extraction index immediately?